pkg/podvolume/backupper_test.go

```diff
@@ -542,6 +560,12 @@ func TestBackupPodVolumes(t *testing.T) {
 
 			require.NoError(t, err)
 
+			if test.mockGetRepositoryType {
+				funcGetRepositoryType = func(string) string { return "" }
```

Comment:
does it not work if we just overwrite the func `getRepositoryType`

Reply:
No, because we only make the case with `mockGetRepositoryType==true` fail; while `funcGetRepositoryType` is a global function, once we change it to the fake function, it stays with it for the following cases, which makes the following cases fail